pkg/volume/hostpath/host_path.go

```diff
@@ -47,9 +47,20 @@ func ProbeVolumePlugins(volumeConfig volume.VolumeConfig) []volume.VolumePlugin
 	}
 }
 
+func FakeProbeVolumePlugins(volumeConfig volume.VolumeConfig) []volume.VolumePlugin {
```

Comment:
I slightly prefer adding a new argument to ProbeVolumePlugins, but this way has fewer code changes.